common/app/common/models/LocationSelector.java

```diff
@@ -48,10 +49,11 @@ public void setCountry(final List<SelectableData> country) {
                     }
                     return selector;
                 }).collect(toList());
+        return (countrySelector.size() > 1) ? countrySelector : emptyList();
```

Comment:
`countrySelector.isEmpty()` == `countrySelector.size() > 0`, not `countrySelector.size() > 1` ;)